controllers/actions.github.com/helpers_test.go

```diff
@@ -0,0 +1,71 @@
+package actionsgithubcom
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+const defaultGitHubToken = "gh_token"
+
+func startManagers(t ginkgo.GinkgoTInterface, managers ...manager.Manager) {
+	for _, mgr := range managers {
```

Comment:
To ensure manager is passed:
```suggestion
func startManagers(t ginkgo.GinkgoTInterface, first, others ...manager.Manager) {
	for _, mgr := range append([]manager.Manager{first}, others...) {
```